charts/minecraft/templates/deployment.yaml

```diff
@@ -266,6 +266,10 @@ spec:
           value: "TRUE"
         {{- end }}
         {{- end }}
+        {{- if .Values.minecraftServer.spigetResources }}
+        - name: SPIGET_RESOURCES
+          value: {{ include "helm-toolkit.utils.joinListWithComma" .Values.minecraftServer.spigetResources }}
```

Comment:
Please use a `join ","`

Reply:
Then should I update [minecraft-proxy](https://github.com/itzg/minecraft-server-charts/blob/2fea96dd82706f6b083b30f5dd9848b10d8b43c6/charts/minecraft-proxy/templates/deployment.yaml#L100) as well?